lib/__tests__/_algoliaAgent.test.ts

```diff
@@ -1,9 +1,11 @@
-import AlgoliaInsights from "../insights";
```

Comment:
naming is confusing here, but not a big deal, personally I'd call the instance of it something like `analyticsInstance`

Reply:
I was confused too at first, then I forgot. Thanks for pointing it out!

https://github.com/algolia/search-insights.js/pull/140/commits/7a151b4adf1327bf9d569879245cb54d4e642064